broadbean/deprecate.py

```diff
@@ -0,0 +1,114 @@
+import types
```

Comment:
shall we give the module a different name from `deprecate` so that it does not clash with the function? :) of course the current naming will mostly work, but perhaps we can make it less confusing for us what is module what is function

Reply:
yes that's a good call. I actually wanted to see what we all think about creating a new package that contains helper modules like this. This code is essentially the same that is in qcodes but I don't think we want broadbean to depend on qcodes due to the circular dependency t would create.

Reply:
> I don't think we want broadbean to depend on qcodes ...

agree, that makes sense to me.


> .... due to the circular dependency t would create

oh, that's fair!

> I actually wanted to see what we all think about creating a new package that contains helper modules like this. This code is essentially the same that is in qcodes 

sure, but to me this is not worth it. this deprecation code is small, and never changes, so it's fine to copy-paste it a few times. making a separate package is extra work, extra potential maintenance, etc.